politeiawww/api/cms/v1/v1.go

```diff
@@ -992,3 +994,37 @@ type CastVoteReply struct {
 	Error           string                 `json:"error"`                 // Error status message
 	ErrorStatus     cmsplugin.ErrorStatusT `json:"errorstatus,omitempty"` // Error status code
 }
+
+// ProposalBillingSummary allows for all proposal spending to be returned for
+// an admin to review.
+type ProposalBillingSummary struct {
```

Comment:
This route needs a max page size policy to serve as the upper bound for the count param.